react/features/base/participants/middleware.js

```diff
@@ -587,6 +661,59 @@ function _raiseHandUpdated({ dispatch, getState }, conference, participantId, ne
     }
 }
 
+/**
+ * Handles a local recording status update.
+ *
+ * @param {Function} dispatch - The Redux dispatch function.
+ * @param {Object} conference - The conference for which we got an update.
+ * @param {string} participantId - The ID of the participant from which we got an update.
+ * @param {boolean} newValue - The new value of the local recording status.
+ * @returns {void}
+ */
+function _localRecordingUpdated({ dispatch, getState }, conference, participantId, newValue) {
+
+    const state = getState();
+
+    dispatch(participantUpdated({
+        conference,
+        id: participantId,
+        localRecording: newValue
```

Comment:
if this is supposed to be a boolean indicating whether this participant is recording or not, it should be reflected in its naming (initially I thought it is holding some object with recording info)